Tests/IntegrationTests.swift

```diff
@@ -94,19 +111,20 @@ final class IntegrationTests: XCTestCase {
 
         let newValue = "text data".data(using: .utf8)
         settings.data = newValue
-        XCTAssertEqual(TestSettings.store.fetch("data"), newValue)
+        XCTAssertEqual(TestSettings.store.data(forKey: "data"), newValue)
 
         settings.data = nil
-        XCTAssertNil(TestSettings.store.fetchOptional("data") as URL?)
```

Comment:
If PR rejected, this typo should be fixed and casted to `Data?` not `URL` otherwise.

Reply:
Opps, thanks! A separate fix would be great here. 😊